src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Status.tsx

```diff
@@ -144,22 +144,22 @@ export const Status = () => {
                 fontWeight='bold'
                 value={bnOrZero(
                   state.withdraw.txStatus === 'pending'
-                    ? state.withdraw.estimatedGasCrypto
+                    ? state.withdraw.withdrawFeeCryptoBaseUnit
                     : state.withdraw.usedGasFee,
```

Comment:
Remove, this would revert to the used gas once confirmed, which isn't the L1/THOR fee

Reply:
derp, tackled it in the deposit PR only - a3c71a501779cde9d4bf82d1b907ff82bff8be2c